### ISSUE TITLE
Nginx reverse proxy

### ISSUE THREAD

**Issue by chrisanthropic** [chrisanthropic/PlexInTheCloud#9]
Set up Nginx with reverse proxy for all web services.

**Comment by zackbcom** [comment]
Let me know if you would like some help on this enhancement.

**Comment by chrisanthropic** [comment]
Definitely! The ones marked 'help wanted' are simply the ones I probably get to on my own.
